src/ui/map.js

```diff
@@ -1062,6 +1062,22 @@ class Map extends Camera {
         this.style.addImage(id, { data: ((data: any): RGBAImage), pixelRatio, sdf });
     }
 
+    /**
+     * Get an image from the style (such as one used by `icon-image` or `background-pattern`).
+     *
+     * @param id The ID of the image.
+     */
+    getImage(id: string): ?StyleImage {
```

Comment:
the build is failing because the type `StyleImage` was not imported so flow is erroring. I think we should rename the method `hasImage` and change the return type to `boolean`